packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -23,7 +21,7 @@
 }
 
 .MafsView .movable-segment {
-    --mafs-segment-stroke-color: var(--mafs-line-color);
```

Comment:
This was the wrong variable for segment stroke, should have been fg (or you can make it offBlack64 like you did)